geo/maxmind.go

```diff
@@ -0,0 +1,122 @@
+package geo
+
+import (
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/oschwald/geoip2-golang"
+)
+
+const (
+	MaxMindUrl     = "https://raw.githubusercontent.com/leo108/geolite2-db/master/Country.mmdb"
```

Comment:
The name of this db is geolite2, not maxmind (maxmind has other geoip db like geoip2)